network/commons/common.go

```diff
@@ -31,6 +31,30 @@ const (
 	topicPrefix = "ssv.v2"
 )
 
+const (
+	signatureSize    = 256
+	signatureOffset  = 0
+	operatorIDSize   = 8
+	operatorIDOffset = signatureOffset + signatureSize
+	messageOffset    = operatorIDOffset + operatorIDSize
+)
+
+func EncodeSignedSSVMessage(message []byte, operatorID spectypes.OperatorID, signature []byte) []byte {
+	return append(append(signature, binary.LittleEndian.AppendUint64(nil, operatorID)...), message...)
```

Comment:
If I'm not wrong, each `append` here allocates another slice.

This should bring down the allocations from 2 to 1:
```suggestion
	b := make([]byte, signatureSize + operatorIDSize + len(message))
	copy(b[signatureOffset:], signature)
	binary.LittleEndian.PutUint64(b[operatorIDOffset:], uint64(operatorID))
	copy(b[messageOffset:], message)
	return b
```

Reply:
done